### ISSUE TITLE
BRS-031/032/033: VR.630

### ISSUE THREAD

**Issue by PerLysemose** [Energinet-DataHub/geh-charges#130]

**Comment by lasrinnil** [comment]
This rule might already be implemented in ChangingTariffVatValueNotAllowedRule.cs

**Comment by prtandrup** [comment]
That's right. I just checked up on this, and we need to make this rule applicable for fees and subscriptions as well
